spec/javascripts/spec/track_click.spec.js

```diff
@@ -24,14 +24,15 @@ describe('A click tracker', function() {
       spyOn(root.GOVUKAdmin, 'trackEvent');
       tracker.start(element);
       element.find("a.foo").click();
-      expect(GOVUKAdmin.trackEvent).toHaveBeenCalledWith('button-pressed', 'Foo');
+
+      expect(GOVUKAdmin.trackEvent).toHaveBeenCalledWith('button-pressed', 'Foo', undefined, undefined);
```

Comment:
Are the `undefined`s necessary here? The `undefined` _value of_ `value` was there before.